package.json

```diff
@@ -26,6 +26,7 @@
     "node": "~0.10.0 || ~0.12.0 || ^4.2.0"
   },
   "dependencies": {
+    "amperize": "https://github.com/jbhannah/amperize#98343579b3a350302c6e4a787ad9ac3faae7d143",
```

Comment:
Do we want to see if the package author would be willing to do a version bump? Don't have any problems with depending on a specific commit, but we would have to make a note sometime to come back and update it to a specific version (greenkeeper doesn't pick up version changes for a specific commit iirc)